docs/pages/web/helpbutton.js

```diff
@@ -0,0 +1,217 @@
+- To provide information or links to information that enhances a user's understanding of the feature.
+        `}
+          />
+
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+- To provide extensive information that is longer than a short sentence. Use [SlimBanner](/web/slimbanner) instead.
+- To display recommendations for how to improve a user's experience. Use [SlimBanner](/web/slimbanner) instead.
+- To display time-sensitive information to a user. Use [Callout](/web/callout) or [SlimBanner](/web/slimbanner) instead.
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <Example
+            id="no-final-version-do"
+            name="How to use"
+            defaultCode={`
+<Flex height="100%" justifyContent="center" alignItems="center" gap={1}>
+  <Checkbox labelDisplay="hidden" label="Created pins checkbox" id="created-pins" onChange={() => {}} />
+  <Text>Pins created in the last 30 days</Text>
+  <HelpButton
+    text="Date range must include at least 1 day in the last 30 days"
+    accessibilityPopoverLabel="Date range must include at least 1 day in the last 30 days"
+  />
+</Flex>
+`}
+          />
+
+          <Example
+            id="no-final-version-dont"
+            name="How to not use"
+            defaultCode={`
+<Text inline>
+  See the total impact of your paid and organic content working together to increase page
+  visits
+  <HelpButton
+    text="Number of times people visited your website after seeing your content on Pinterest"
+    accessibilityPopoverLabel="Number of times people visited your website after seeing your content on Pinterest"
+  />
+  {' '}add-to-carts{' '}
+  <HelpButton
+    text="Number of times people added your products to their carts after interacting with your Pins"
+    accessibilityPopoverLabel="Number of times people added your products to their carts after interacting with your Pins"
+  />
+  {' '}and checkouts
+  <HelpButton
+    text="Number of checkouts stemming from your Pins and ads"
+    accessibilityPopoverLabel="Number of checkouts stemming from your Pins and ads"
+  />
+  .
+</Text>
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Place HelpButton to the end of the element you wish to add guidance to."
+            sandpackExample={
+              <SandpackExample code={endGuideElement} name="End of guide element" hideEditor />
+            }
+          />
+
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Do not include HelpButton within blocks of content."
```

Comment:
For the "Do" and "Don't" cards, we assume that each point is prefaced with "Do" or "Don't". So "Do not" is not needed here — the sentence should start with "Include HelpButton within…"